tests/DependencyInjection/MetaModelsFilterPerimetersearchExtensionTest.php

```diff
@@ -28,6 +29,7 @@
 
 /**
  * This test case test the extension.
+ * @covers \MetaModels\FilterPerimetersearchBundle\DependencyInjection\MetaModelsAttributeRatingExtension
```

Comment:
Please add a line break here (as for all other tests).

Reply:
done

Reply:
Would you please push the change then? It is not here on github yet.

Reply:
pls check now